sdk/servicebus/test-resources.json

```diff
@@ -193,47 +171,47 @@
     }
   ],
   "outputs": {
-    "SERVICE_BUS_CONNECTION_STR": {
+    "SERVICEBUS_CONNECTION_STR": {
```

Comment:
reason for updating these? I think you may need to update the envvars in the samples as well, since this template is used for running the samples.

Reply:
The prefix calculated by the deployment script is `SERVICEBUS` so the tests should be consistent with that prefix which would simplify common lookup handlers (https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1679600&view=logs&j=7409dde9-db2c-5b85-4a98-75043a08df8b&t=888866e9-3e1f-5697-2921-9cd89d42e7b0&l=3344). But I agree checking to make sure there are no references to `SERVICE_BUS` would good if not already done.